### PR TITLE
Add more then one input placeholder in the compose view

### PR DESCRIPTION
**Pull request by sahil-sahu-dev**
-> Added two placeholder texts- 
     1. Express what's on your mind
     2. Tell us what you think

-> Changed the compose view to randomly pick one of the three placeholder strings.
-> Added string localizations for all supported languages

### CODE REVIEW

File: Mastodon.xcodeproj/xcshareddata/xcschemes/Mastodon.xcscheme [hunk 1]
```diff
@@ -73,6 +73,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "de"
```
Comment (zeitschlag) [inline Mastodon.xcodeproj/xcshareddata/xcschemes/Mastodon.xcscheme]: This runs the app in German. To me (personally), this is not an issue at all, but I doubt, that every other contributor speaks German :)

File: MastodonSDK/Sources/MastodonLocalization/Resources/ar.lproj/Localizable.strings [hunk 2]
```diff
@@ -193,6 +193,8 @@
 "Scene.Compose.AutoComplete.SpaceToAdd" = "انقر على مساحة لإضافتِها";
 "Scene.Compose.ComposeAction" = "نَشر";
 "Scene.Compose.ContentInputPlaceholder" = "عَبِّر عَمَّ يَجُولُ فِي ذِهنِك";
+"Scene.Compose.ContentInputPlaceholder2" = "عبر عما يدور في ذهنك";
+"Scene.Compose.ContentInputPlaceholder3" =  "اخبرنا ماذا تعتقد";
```
Comment (zeitschlag) [inline MastodonSDK/Sources/MastodonLocalization/Resources/ar.lproj/Localizable.strings:197]: All those changes will be overriden when there's a new update with updated translations from Crowdin. Please have a look at the [Localization-folder](https://github.com/mastodon/mastodon-ios/tree/develop/Localization), there's a short description :)

tl;dr: Please add your placeholders to [`Localization/app.json`](https://github.com/mastodon/mastodon-ios/blob/develop/Localization/app.json) and to  [`Base.lproj/Localizable.strings`](https://github.com/mastodon/mastodon-ios/pull/867/files#diff-451756534c4f3dc8961715f07e2573b5f71e8586a2794d373475f02ae5c9443f)` (which you already did 🙂 ) . Translation itself is done on Crowdin

File: MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/View/ComposeContentView.swift [hunk 3]
```diff
@@ -23,6 +23,7 @@ public struct ComposeContentView: View {
     
     @ObservedObject var viewModel: ComposeContentViewModel
     
+    private let randomContentInputPlaceholderChoice = Int.random(in: 1...3)
```
Comment (zeitschlag) [inline MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/View/ComposeContentView.swift]: What do you think about something like this? 

```swift
private let var randomContentInputPlaceholder = [
    L10n.Scene.Compose.contentInputPlaceholder,
    L10n.Scene.Compose.contentInputPlaceholder2
    L10n.Scene.Compose.contentInputPlaceholder3,
].randomElement()
```

It would make it easier to add new additional placeholders.

**Review by zeitschlag — CHANGES_REQUESTED** [review]
Thank you for your work @sahil-sahu-dev 🥳 It feels like having a nice, little easter-egg in the app.

I added some comments, please have a look at them 🙂 Can you also please take care of the merge-conflicts? Thank you!

File: MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/View/ComposeContentView.swift (hunk 3 above)
Comment (j-f1) [reply MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/View/ComposeContentView.swift]: to make sure the placeholder stays consistent even if the `ComposeContentView` is recreated:

```swift
@State private var randomContentInputPlaceholder = [
    L10n.Scene.Compose.contentInputPlaceholder,
    L10n.Scene.Compose.contentInputPlaceholder2
    L10n.Scene.Compose.contentInputPlaceholder3,
].randomElement()
```

File: Mastodon.xcodeproj/xcshareddata/xcschemes/Mastodon.xcscheme (hunk 1 above)
Comment (sahil-sahu-dev) [reply Mastodon.xcodeproj/xcshareddata/xcschemes/Mastodon.xcscheme]: Ah! I did that to test out the feature in another language. I'll change that

File: MastodonSDK/Sources/MastodonLocalization/Resources/ar.lproj/Localizable.strings (hunk 2 above)
Comment (sahil-sahu-dev) [reply MastodonSDK/Sources/MastodonLocalization/Resources/ar.lproj/Localizable.strings:197]: I see, I'll do that

File: MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/View/ComposeContentView.swift (hunk 3 above)
Comment (sahil-sahu-dev) [reply MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/View/ComposeContentView.swift]: That's a much cleaner implementation. I like how it keeps the choices neatly together. Great suggestion!

File: MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/View/ComposeContentView.swift [hunk 4]
```diff
@@ -23,6 +23,11 @@ public struct ComposeContentView: View {
     
     @ObservedObject var viewModel: ComposeContentViewModel
     
+    @State private var randomContentInputPlaceholder = [
+        L10n.Scene.Compose.contentInputPlaceholder,
+        L10n.Scene.Compose.contentInputPlaceholder2,
+        L10n.Scene.Compose.contentInputPlaceholder3,
+    ].randomElement()
```
Comment (j-f1) [inline MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/View/ComposeContentView.swift:30]: You should be safe to force unwrap this since `randomElement` only returns nil if the collection is empty ([docs](https://developer.apple.com/documentation/swift/array/randomelement(using:))).

Comment (sahil-sahu-dev) [reply MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/View/ComposeContentView.swift:30]: Yup, that makes sense. Done!
